### PR TITLE
logging and remove spurious print stmts

### PR DESCRIPTION
**Pull request by julian1**

### CODE REVIEW

**Comment by jonescc** [comment]
I''l review

File: src/main/java/au/org/emii/ncdfgenerator/NcdfEncoder.java [hunk 1]
```diff
@@ -228,10 +224,9 @@ else if( value instanceof Array )
 		}
 		// should we log here, or allow exceptions to propagate up?
 		catch ( Exception e ) {
-			// TODO log
 			StringWriter sw = new StringWriter();
 			e.printStackTrace(new PrintWriter(sw));
-			System.out.println( "Opps " + sw.toString() );
+			logger.error( "Problem generating netcdf " + sw.toString() );
```
Comment (jonescc) [inline src/main/java/au/org/emii/ncdfgenerator/NcdfEncoder.java]: Logger.error(String msg, Throwable t) would remove the need to use a stringwriter

File: src/main/java/au/org/emii/ncdfgenerator/NcdfEncoder.java [hunk 2]
```diff
@@ -126,8 +124,7 @@ public InputStream get() throws Exception
 					" order by " + orderClause +
 					";" ;
 
-				// TODO logger
-				System.out.println( "instanceId is " + instanceId + " " + query);
+				logger.info( "instanceId " + instanceId + ", " + query );
```
Comment (jonescc) [inline src/main/java/au/org/emii/ncdfgenerator/NcdfEncoder.java]: Might be a little chatty for the prod web service logs.  Perhaps use debug and change resources/log4j.properties to report debug level during testing while your still working on this

**Comment by jonescc** [comment]
Finished review.  Some minor comments only.

**Comment by julian1** [comment]
Fixed

**Comment by julian1** [comment]
And rethrow

**Comment by jonescc** [comment]
merge conflicts

**Comment by julian1** [comment]
https://github.com/aodn/ncdfgenerator/pull/25

**Pull request closed without merging**